mmv1/products/tpuv2/Vm.yaml

```diff
@@ -88,3 +94,220 @@ properties:
+        name: 'email'
+        default_from_api: true
+        immutable: true
+        description: |
+          Email address of the service account. If empty, default Compute service account will be used.
+      - !ruby/object:Api::Type::Array
+        name: 'scope'
+        default_from_api: true
+        immutable: true
+        diff_suppress_func: 'tpuServiceAccountAddedScopesSuppress'
+        description: |
+          The list of scopes to be made available for this service account. If empty, access to all
+          Cloud APIs will be allowed.
+        item_type: Api::Type::String
+  - !ruby/object:Api::Type::NestedObject
+    name: 'schedulingConfig'
+    description: |
+      The scheduling options for this node.
+    immutable: true
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: 'preemptible'
+        immutable: true
+        send_empty_value: true
+        description: |
+          Defines whether the node is preemptible.
+      - !ruby/object:Api::Type::Boolean
+        name: 'reserved'
+        immutable: true
+        send_empty_value: true
+        description: |
+          Whether the node is created under a reservation.
+  - !ruby/object:Api::Type::Array
+    name: 'dataDisks'
+    description: |
+      The additional data disks for the Node.
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'sourceDisk'
+          required: true
+          description: |
+            Specifies the full path to an existing disk. For example:
+            "projects/my-project/zones/us-central1-c/disks/my-disk".
+        - !ruby/object:Api::Type::Enum
+          name: 'mode'
+          description: |
+            The mode in which to attach this disk. If not specified, the default is READ_WRITE
+            mode. Only applicable to dataDisks.
+          values:
+            - 'READ_WRITE'
+            - 'READ_ONLY'
+  - !ruby/object:Api::Type::NestedObject
+    name: 'shieldedInstanceConfig'
+    description: |
+      Shielded Instance options.
+    immutable: true
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: 'enableSecureBoot'
+        immutable: true
+        send_empty_value: true
+        required: true
+        description: |
+          Defines whether the instance has Secure Boot enabled.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: Resource labels to represent user-provided metadata.
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'metadata'
+    description: Custom metadata to apply to the TPU Node. Can set startup-script and shutdown-script.
+  - !ruby/object:Api::Type::Array
+    name: 'tags'
+    description: |
+      Tags to apply to the TPU Node. Tags are used to identify valid sources or targets for network firewalls.
+    item_type: Api::Type::String
+  - !ruby/object:Api::Type::String
+    name: 'state'
+    output: true
```

Comment:
Yep, `output: true` is our equivalent to output-only; it cannot be set by the user. And yea, this is where we specify that for mmv1 resources.

Reply:
I see. It didn't seem required to state the outputs when I tested it and I didn't see this in some of the other resource files I checked. This seems fine, just curious if this is the preferred method since I'm not sure if I saw any difference when output fields were omitted.

Reply:
Yea, you won't typically see a difference in output fields when testing, but we include them to better support users. They can feed outputs into other config or automation they may be using.